### PR TITLE
Fix199

### LINKED ISSUE #199: Remove dependency on UCSCtools/samtools
Now that deepTools uses pyBigWig, there's no need for UCSCtools. While not yet completely documented, pyBigWig now supports bigWig file creation without intermediate files. Also, there's nothing that samtools does that we can't just do with pysam.

These two changes would considerably simplify the installation as well.

### PR DESCRIPTION
**Pull request by dpryan79**
This fixes #199 but will only work once pyBigWig 2.0 is released. My minimal testing suggests that this should work, but we should really test further before merging. Thus, this will be a WIP for a while.

### CODE REVIEW

**Comment by bgruening** [comment]
Ok, for what it's worth ... all or nothing I would say. We depend on this new version, rerunning all our tests and doing the release. For the Galaxy Servers this feature is not essential, but for commandline folks this is nice to have. So if it does not break old stuff, let's merge it before releasing. Just my 5c.

File: deeptools/correctGCBias.py [hunk 1]
```diff
@@ -617,17 +593,18 @@ def main(args=None):
...
-                                                " ".join(res))
+            print "concatenating (sorted) intermediate BAMs"
+            of = pysam.open(args.correctedFile.name, "wb", template=res[0])
+            for f in res:
+                for e in f.fetch(until_eof=True):
+                    of.write(e)
+            of.close()
 
-        run_shell_command(command)
+        print "indexing BAM"
```
Comment (bgruening) [inline deeptools/correctGCBias.py]: For 2.1 we should get rid of all print statements and use logging.

**Comment by bgruening** [comment]
Nice cleanup! :+1:  from my side.

File: deeptools/correctGCBias.py (hunk 1 above)
Comment (dpryan79) [inline deeptools/correctGCBias.py]: Is there really any benefit to using logging for stuff like this?

**Comment by dpryan79** [comment]
We'll be testing this locally for the next few days to hopefully catch any introduced bugs.

**Pull request merged**